data/alexandria start.txt

```diff
@@ -0,0 +1,1304 @@
+			choice
+				`	Interrupt him...`
+			`The man turns to you and says, "it is always an honor to meet Alexandria Seekers.  How can I help you?"`
+			`	Daemos responds, "I seek the Paq cha'DIch, second volume of the Chal Jub Paqmey.  I've heard its your kind of literature.  Do you have it?"`
+			`	"I don't have it now, but I can get it for you.  Do you have the book, 'A Thousand Poisons of Amazon and How to Conceal Their Taste?'  I've been looking for a signed first edition."`
+			`	"Ekhh.  A trade.  Okay, I'll get you a copy."`
+			`	On the way out, Daemos whispers to you, "we have to fly to planet Amazon in the Schedar system next.  I have the book, but not the signature.`
+			`	"Meet me in the spaceport when you're ready to go."`
+
+
+
+
+mission "Alexandria Library Card: Second Volume: Step 2"
+	name "Alexandria Library Card"
+	description "Take Dameos Titan to <destination> to search for the second of the five Chal Jub Paqmey."
+	blocked `Daemos complains, "I can't sleep in cooling ducts.  I need a bunk and space for 1 ton of cargo."`
+	source Tinker
+	destination Amazon
+	passengers 1
+	cargo "old books and a thousand pencils" 1
+	to offer
+		has "Alexandria Library Card: Second Volume: Step 1: done"
+	on offer
+		conversation
+			`Daemos meets you at your ship and says, "our destination is in a dangerous area of Syndicate space.  I've arranged some escorts so you don't get me killed."`
+			label questions
+			choice
+				`	"What will we do on Amazon?"`
+					goto what
+				`	"Why do we need escorts?"`
+					goto why
+				`	"Sounds good.  Let's go."`
+					goto go
+				`	"I don't have time to help you any more."`
+					decline
+			label what
+			`"We need to get a first edition book signed.  It's a popular book among the Syndicate, so I always keep a copy or two when I visit their aristocracy.  I'm a personal friend of the author, and I know the location of his remote hermitage."`
+				goto questions
+			label why
+			`Daemos responds, "that area is crawling with pirates and I doubt your ship will survive alone.  The Chal Jub Paqmey are critical, so the Library is willing to expend significant resources to obtain it."`
+			`	"Why am I your courier if you fear I'll get you killed?"`
+			`	"Courier?  You're not a courier.  I don't understand."`
+			`	"You're paying me to ferry you around the galaxy looking for books."`
+			`	"No.  You don't understand.  You're my apprentice.  Alexandria Library doesn't ask just anyone to assist a Grandmaster Seeker.  They think you have potential to be a Seeker, despite all your faults.  The payments are just stipends to keep you fed during your apprenticeship."`
+			label "seeker questions"
+			choice
+				`	"What is a Seeker?"`
+					goto seeker
+				`	"Why me?"`
+					goto "why me"
+				`	"What faults?"`
+					goto faults
+				`	Change the subject...`
+					goto questions
+				`	"Sounds good.  Let's go."`
+					goto go
+				`	"I don't have time to help you any more."`
```

Comment:
A fine suggestion.  I like it.

Reply:
Actually, it doesn't work since the player won't know about the Seeker aspect unless they choose the right dialog options.  This makes me miss the `[show_if]` feature in WML.  There is no way have a conditionally-shown choice.